src/object.c

```diff
@@ -237,9 +279,14 @@ int object_load_packed(SDL_RWops* rw)
 	}
 
 	if (object_calculate_checksum(&entry, chunk, chunkSize) != entry.checksum){
-		log_error("Checksum mismatch from packed object: %.8s", entry.name);
-		free(chunk);
-		return 0;
+	
+		if(gConfigGeneral.allow_loading_with_incorrect_checksum){
```

Comment:
After `if` and before the curly bracket, i.e.
```c
if (gConfigGeneral.allow_loading_with_incorrect_checksum) {
```

Same thing in [line 215](https://github.com/OpenRCT2/OpenRCT2/pull/3021/files#diff-43278b647ec38f9faf284496e22a97d5R215)